backbone.ipynb

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
<p>Can you add a comment for the logic in line 74-76? Looks like we're string parsing the layer parameter prefixed on the training layers to disable gradient computation. Wasn't super clear on first sight.</p><p> </p><p>Furthermore, I'm not sure if <code>requires_grad_</code> is all we need to do. What about <code>BatchNorm</code> statistics? Will they be frozen as well?</p><p> </p><p>*UPDATE:* ^^ Realized we're explicitly using <code>FrozenBatchNorm2d</code>  for this purpose, NVM.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yycho0108/ai604-video-object-pose/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>